browser/ephemeral_storage/ephemeral_storage_browsertest.cc

```diff
@@ -253,41 +259,72 @@ IN_PROC_BROWSER_TEST_F(EphemeralStorageBrowserTest, StorageIsPartitioned) {
 }
 
 IN_PROC_BROWSER_TEST_F(EphemeralStorageBrowserTest,
-                       NavigatingClearsEphemeralStorage) {
+                       NavigatingClearsEphemeralStorageAfterKeepAlive) {
   ui_test_utils::NavigateToURL(browser(), a_site_ephemeral_storage_url_);
   auto* web_contents = browser()->tab_strip_model()->GetActiveWebContents();
+  ASSERT_TRUE(WaitForLoadStop(web_contents));
 
   SetValuesInFrames(web_contents, "a.com value", "from=a.com");
 
-  ValuesFromFrames values_before = GetValuesFromFrames(web_contents);
-  EXPECT_EQ("a.com value", values_before.main_frame.local_storage);
-  EXPECT_EQ("a.com value", values_before.iframe_1.local_storage);
-  EXPECT_EQ("a.com value", values_before.iframe_2.local_storage);
+  ValuesFromFrames values = GetValuesFromFrames(web_contents);
+  EXPECT_EQ("a.com value", values.main_frame.local_storage);
+  EXPECT_EQ("a.com value", values.iframe_1.local_storage);
+  EXPECT_EQ("a.com value", values.iframe_2.local_storage);
 
-  EXPECT_EQ("a.com value", values_before.main_frame.session_storage);
-  EXPECT_EQ("a.com value", values_before.iframe_1.session_storage);
-  EXPECT_EQ("a.com value", values_before.iframe_2.session_storage);
+  EXPECT_EQ("a.com value", values.main_frame.session_storage);
+  EXPECT_EQ("a.com value", values.iframe_1.session_storage);
+  EXPECT_EQ("a.com value", values.iframe_2.session_storage);
 
-  EXPECT_EQ("from=a.com", values_before.main_frame.cookies);
-  EXPECT_EQ("from=a.com", values_before.iframe_1.cookies);
-  EXPECT_EQ("from=a.com", values_before.iframe_2.cookies);
+  EXPECT_EQ("from=a.com", values.main_frame.cookies);
+  EXPECT_EQ("from=a.com", values.iframe_1.cookies);
+  EXPECT_EQ("from=a.com", values.iframe_2.cookies);
 
   // Navigate away and then navigate back to the original site.
   ui_test_utils::NavigateToURL(browser(), b_site_ephemeral_storage_url_);
+  ASSERT_TRUE(WaitForLoadStop(web_contents));
```

Comment:
you don't need these, `ui_test_utils::NavigateToURL` blocks until loading stops on its own

Reply:
yea, I added this because I was seeing strange behavior and then forgot to remove it